spec/intl.html

```diff
@@ -922,7 +935,8 @@ <h1>HandleDateTimeTemporalZonedDateTime ( _dateTimeFormat_, _zonedDateTime_ )</h
       1. Return the Record {
           [[pattern]]: _pattern_.[[pattern]],
           [[rangePatterns]]: _pattern_.[[rangePatterns]],
-          [[epochNanoseconds]]: _instant_.[[Nanoseconds]]
+          [[epochNanoseconds]]: _instant_.[[Nanoseconds]],
```

Comment:
_(leaving a comment here because I can't leave it on line 918 above)_ 

I think this spec text above may be a problem: 

> 1. If _dateTimeFormat_.[[TimeZone]] is not equal to DefaultTimeZone(), and _timeZone_ is not equal to _dateTimeFormat_.[[TimeZone]], then
>   1. Throw a *RangeError* exception.

If I'm reading this correctly, the code below would not throw when the DTF's `timeZone` option matches the system's time zone.  Based on my understanding, this is not desired behavior. @ptomato @sffc is this a correct reading?

```js
// run this code on a computer with system time zone America/Los_Angeles
zdt = Temporal.ZonedDateTime.from('2020-01-01[Asia/Tokyo');

new Intl.DateTimeFormat('en').format(zdt);
// => as expected, formats using ZDT time zone Asia/Tokyo

new Intl.DateTimeFormat('en', { timeZone: 'America/Denver' }).format(zdt);
// => as expected, RangeError because timeZone option is disallowed when formatting ZDT

new Intl.DateTimeFormat('en', { timeZone: 'America/Los_Angeles' }).format(zdt);
// expected: RangeError because timeZone option is disallowed when formatting ZDT
// actual: no exception, uses ZDT's time zone because time zone option matches DefaultTimeZone()
```

Reply:
Yeah, we'd need to change those lines above. If we go with my suggestion from a previous comment, I think we can remove them altogether because the check would be done in FormatDateTimePattern?